zkevm-circuits/src/evm_circuit/step.rs

```diff
@@ -177,8 +177,9 @@ impl From<&ExecError> for ExecutionState {
             ExecError::InvalidCreationCode => ExecutionState::ErrorInvalidCreationCode,
             ExecError::InvalidJump => ExecutionState::ErrorInvalidJump,
             ExecError::ReturnDataOutOfBounds => ExecutionState::ErrorReturnDataOutOfBound,
-            ExecError::CodeStoreOutOfGas => ExecutionState::ErrorOutOfGasCodeStore,
-            ExecError::MaxCodeSizeExceeded => ExecutionState::ErrorMaxCodeSizeExceeded,
+            ExecError::CodeStoreOutOfGas | ExecError::MaxCodeSizeExceeded => {
```

Comment:
Have a question, do we need to list `ExecError::CodeStoreOutOfGas` here? I thought it's covered in L193 (`OogError::CodeStore => ExecutionState::ErrorCodeStore`)